### PR TITLE
ANDROID-11118 Allowing small images in rows

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
Allowing also small images in the Sheet rows.

### :test_tube: How can I test this?
Adding a new class: `AssetViewData.SmallImage`

https://user-images.githubusercontent.com/4595241/191953823-24e241c9-68f3-412d-9f43-02a0c426da30.mp4

### CODE REVIEW

**Review by DevPabloGarcia — APPROVED** [review]
Nice !

File: library/src/main/java/com/telefonica/mistica/bottomsheet/children/list/SelectableListAdapter.kt [hunk 1]
```diff
@@ -76,6 +76,10 @@ private fun ListRowView.setAsset(asset: AssetViewData?) {
             this.setAssetResource(asset.id)
         }
         null -> {}
+        is AssetViewData.SmallImage -> {
+            this.setAssetType(TYPE_SMALL_ICON)
+            this.setAssetDrawable(asset.drawableRes)
+        }
```
Comment (jeslat) [inline library/src/main/java/com/telefonica/mistica/bottomsheet/children/list/SelectableListAdapter.kt:82]: Really minor, but I would put it before the null case

**Review by jeslat — APPROVED** [review]
good job!

File: library/src/main/java/com/telefonica/mistica/bottomsheet/children/list/SelectableListAdapter.kt (hunk 1 above)
Comment (gmerinojimenez) [reply library/src/main/java/com/telefonica/mistica/bottomsheet/children/list/SelectableListAdapter.kt:82]: totally

**Review by yceballost — APPROVED** [review]
nice!!

**Pull request merged**